index.html

```diff
@@ -8,41 +8,89 @@
 <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 </head>
 <body>
-  <header>
+  <header class="header-m">
     <div class="logo">
       <img  src="./images/image_geometry_1.png" alt="logo image">
    </div>
     <nav>
       <div class="menu">
            <i class="fa fa-bars"></i>
       </div>
+        <ul class="menu-d-list">
+          <li>Hello <hr></li>
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+      
     </nav>
   </header>
   <main>
   
     <section class="headline"> 
+      <div class="headline-d">
+
         <h1 class="myname">Charlie 
         Giorgania
         </h1> 
-        <p  class="headline-p">Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. <br> <br>
+
+        <p class="headline-p">Hello! I am a software developer! I can help you build a product, 
+          feature or website. Take a look at my work. <br> <br>
   
           If you like what you see and have a project you need coded, don’t hesitate to contact me.
         </p>
-      <button class="headline-btn">
-        Start collaboration
-      </button>
+
+        <p class="headline-p-d">Hello! I am a software developer! I can help you build a product, 
+          feature or website. Take a look at my work. <br> <br>
+  
+          If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <button class="headline-btn">
+          Start collaboration
+        </button>
+
+      </div>
+
       <div class="headline-img">
-       
+        
       </div>
-  </section>
- <section class="projects">
+      <div class="image-d">
+        <img class="img1-d"src="./images/image_dots.png" alt="dotd image">
+
+        <img class="img2-d" src="./images/image-geometry_1.png" alt="introimg">
+    </div>
+
+    </section>
+  <section class="projects">
+        <div class="project-d-header">
           <h2 class="project-title">Projects</h2>
+          
+          <div class="project-card-d">
+              <div class= "img-d"><img class= "img3-d" src="./images/image_dots.png" alt="dotd image"></div>
+              <div class="project1-card-d">  
+                  <div class="project1-img-d">
+                  </div>
+                  <div class="project-des-d">
+                      <h2 class="sub-title-d">Project name goes here</h2>
+                      <ul>
+                        <li>HTML/CSS</li>
+                        <li>Ruby on Rails</li>
+                        <li>Javascript</li>
+                      </ul>
+                      <div class="btn-project">
+                      <button>See this  project   <i class="fa fa-long-arrow-right"></i>  </button>
+                      </div> 
+                  </div>
+              </div> 
+          </div>
+        </div>
```

Comment:
Well done on the project section! 👍 However, upon reviewing the mobile version, I noticed a couple of areas that require attention to align with the design.

Firstly, it seems that you missed adding the projects section title. Please ensure you include the section title as per the design specifications.

Additionally, while you utilized grid and flexbox to align the elements, there are still some adjustments needed. In particular, the tag list within the section does not have proper padding. I recommend adding the appropriate padding to achieve the desired alignment, ensuring it matches the design specifications.
![image](https://github.com/hydaline-code/Portfolio_mobilefirst/assets/55840999/f277659c-597c-49a0-835e-a6071b1a371c)
Taking care of these details is important to maintain consistency with the design and create a polished user experience. Keep up the good work! 😄